NaReguaApp/src/routes/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View, ActivityIndicator } from 'react-native';
+
+import { useAuth } from '../contexts/auth';
+
+import AppClienteRoute from './app-cliente.routes';
+import AppPrestadorRoute from './app-prestator.routes';
+import AuthRoute from './auth.routes';
+
+function Routes() {
+	const { logado, isProvider, loading } = useAuth();
+
+	if (loading) {
+		return (
+			<View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+				<ActivityIndicator size="large" color="#666" />
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹↹` with `········`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
        <ActivityIndicator size="large" color="#666" />
```